src/hashes.txt

```diff
@@ -18,528 +18,553 @@ e4a76dd29932125da92a176a7e182d34eb98f65d00640bddae98770d3a28fa97 backbone.localS
-828cbbcacb430f9c5b5d27fe9302f8795eb338f2421010f5141882125226f94f jquery.2.1.3.jquery.js
+140ff438eaaede046f1ceba27579d16dc980595709391873fa9bf74d7dbe53ac jquery.2.1.1.jquery.js
+07cb07bdfba40ceff869b329eb48eeede41740ba6ce833dd3830bd0af49e4898 jquery.2.1.2.jquery.js
+243f6ee513637db6d897f01b89862f54f29c2cd94a35edaead432e1b334421c9 jquery.2.0.1.jquery.min.js
+82f420005cd31fab6b4ab016a07d623e8f5773de90c526777de5ba91e9be3b4d jquery.2.2.1.jquery.min.js
 b2215cce5830e2350b9d420271d9bd82340f664c3f60f0ea850f7e9c0392704e jquery.2.1.4.jquery.js
-2ecd295d295bec062cedebe177e54b9d6b19fc0a841dc5c178c654c9ccff09c0 jquery.1.11.2.jquery.min.js
-4837f7e1f1565ff667528cd75c41f401e07e229de1bd1b232f0a7a40d4c46f79 jquery.1.10.1.jquery.min.js
-7bd80d06c01c0340c1b9159b9b4a197db882ca18cbac8e9b9aa025e68f998d40 jquery.1.9.1.jquery.js
-5f1ab65fe2ad6b381a1ae036716475bf78c9b2e309528cf22170c1ddeefddcbf jquery.1.12.0.jquery.min.js
 8a102873a33f24f7eb22221e6b23c4f718e29f85168ecc769a35bfaed9b12cce jquery.2.2.0.jquery.min.js
-c85537acad72f0d7d409dfc1e2d2daa59032f71d29642a8b64b9852f70166fbb jquery.1.12.0.jquery.js
-2359d383bf2d4ab65ebf7923bdf74ce40e4093f6e58251b395a64034b3c39772 jquery.1.12.1.jquery.min.js
-82f420005cd31fab6b4ab016a07d623e8f5773de90c526777de5ba91e9be3b4d jquery.2.2.1.jquery.min.js
-a18aa92dea997bd71eb540d5f931620591e9dee27e5f817978bb385bab924d21 jquery.2.2.0.jquery.js
-07cb07bdfba40ceff869b329eb48eeede41740ba6ce833dd3830bd0af49e4898 jquery.2.1.2.jquery.js
-78d714ccede3b2fd179492ef7851246c1f1b03bfc2ae83693559375e99a7c077 jquery.2.2.1.jquery.js
-56e843a66b2bf7188ac2f4c81df61608843ce144bd5aa66c2df4783fba85e8ef jquery.1.12.1.jquery.js
-5540b2af46570795610626e8d8391356176ca639b1520c4319a2d0c7ba9bef16 jquery.1.12.2.jquery.js
 6b6de0d4db7876d1183a3edb47ebd3bbbf93f153f5de1ba6645049348628109a jquery.2.2.3.jquery.min.js
-69a3831c082fc105b56c53865cc797fa90b83d920fb2f9f6875b00ad83a18174 jquery.1.12.3.jquery.min.js
-d5732912d03878a5cd3695dc275a6630fb3c255fa7c0b744ab08897824049327 jquery.1.12.3.jquery.js
-dfa729d82a3effadab1000181cb99108f232721e3b0af74cfae4c12704b35a32 jquery.2.2.2.jquery.min.js
+828cbbcacb430f9c5b5d27fe9302f8795eb338f2421010f5141882125226f94f jquery.2.1.3.jquery.js
 e3fcd40aa8aad24ab1859232a781b41a4f803ad089b18d53034d24e4296c6581 jquery.2.2.2.jquery.js
 95a5d6b46c9da70a89f0903e5fdc769a2c266a22a19fcb5598e5448a044db4fe jquery.2.2.3.jquery.js
-95914789b5f3307a3718679e867d61b9d4c03f749cd2e2970570331d7d6c8ed9 jquery.1.12.2.jquery.min.js
+dfa729d82a3effadab1000181cb99108f232721e3b0af74cfae4c12704b35a32 jquery.2.2.2.jquery.min.js
+430f36f9b5f21aae8cc9dca6a81c4d3d84da5175eaedcf2fdc2c226302cb3575 jquery.1.12.4.jquery.js
+896e379d334cf0b16c78d9962a1579147156d4a72355032fce0de5f673d4e287 jquery.2.0.0.jquery.js
+78d714ccede3b2fd179492ef7851246c1f1b03bfc2ae83693559375e99a7c077 jquery.2.2.1.jquery.js
+3fba317398283773b831fad6ea8e20829a36dc65a920b8696db23408a69fb40a jquery.2.1.0-beta2.jquery.min.js
```

Comment:
You'll need to remove these manually; npm seems to return them and I have to remove them but haven't got to that yet: #156